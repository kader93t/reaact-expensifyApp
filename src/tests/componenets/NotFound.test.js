import React from 'react';
import NotFound from "./../../components/NotFoundPage";
import { shallow } from "enzyme";

test('should render the not found page', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
})
