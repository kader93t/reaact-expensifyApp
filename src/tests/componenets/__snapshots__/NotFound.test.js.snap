// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the not found page 1`] = `
<div>
  <h1>
    Not found 404  
  </h1>
  <Link
    to="/"
  >
    <h2>
      Go Home
    </h2>
  </Link>
</div>
`;
